remix-solidity/src/compiler/compiler.js

```diff
@@ -94,7 +100,7 @@ function Compiler (handleImportCall) {
 
         var result
         try {
-          var input = compilerInput(source.sources, {optimize: optimize, evmVersion: evmVersion, target: source.target})
+          var input = compilerInput(source.sources, {optimize: optimize, evmVersion: evmVersion, language: language, target: source.target})
```

Comment:
For making it easier to read you could do : `{ optimize, evmVersion, language, taget: source.target }`